config/routes.rb

```diff
@@ -1,8 +1,14 @@
 Rails.application.routes.draw do
 
   resources :companies do
-    resources :jobs
+    resources :jobs, controller: "companies/jobs"
   end
+
+  resources :categories
+  resources :comments, only: [:create]
```

Comment:
we already talked a little about how nesting a few of these routes might be helpful 👍